adapters/repos/schema/store.go

```diff
@@ -189,8 +189,8 @@ func (r *store) migrate(filePath string, from, to int) (err error) {
 	return nil
 }
 
-// saveSchemaV1 might be needed to migrate from v2 to v0
-func (r *store) saveSchemaV1(schema ucs.State) error {
+// saveSchemaV2 might be needed to migrate from v2 to v1
+func (r *store) saveSchemaV2(schema ucs.State) error {
```

Comment:
I think this is saveSchemaV1(), as it's responsible for migrating from v0 to v1.
The same applies to r.loadSchemaV1()